### PR TITLE
feat: Machine translations & Translation memory TG-431

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

File: docker/base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+# An additional setting that is recommended for all users regardless of this
+# value is the runtime "--stop-timeout" (or your orchestrator/runtime's
+# equivalent) for controlling how long to wait between sending the defined
+# STOPSIGNAL and sending SIGKILL (which is likely to cause data corruption).
+#
+# The default in most runtimes (such as Docker) is 10 seconds, and the
+# documentation at https://www.postgresql.org/docs/12/server-start.html notes
+# that even 90 seconds may not be long enough in many instances.
+
+EXPOSE 5432
```
Comment (stepan662) [inline docker/base/Dockerfile:190]: This is really ugly, it's just copy-paste?

File: docker/base/postgres-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+			echo
+		fi
+	fi
+
+	exec "$@"
+}
+
+if ! _is_sourced; then
+	_main "$@"
+fi
```
Comment (stepan662) [inline docker/base/postgres-entrypoint.sh:349]: Also really ugly

File: docker/base/Dockerfile (hunk 1 above)
Comment (JanCizmar) [reply docker/base/Dockerfile:190]: Yep, it's from their Docker image.

**Review by stepan662 — APPROVED** [review]

File: docker/base/postgres-entrypoint.sh (hunk 2 above)
Comment (JanCizmar) [reply docker/base/postgres-entrypoint.sh:349]: As well as this.

**Pull request merged**